src/styles/style.css

```
@@ -0,0 +1,115 @@
+@import '../styles/stars.css';
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  --white: #FFF;
+  --dark-blue: #060B35;
+  --gray: #616070;
+  --blue: #00ACDC;
+}
+
+.card {
+  width: 198px;
+  border-radius: 5px;
```

add padding for all card sides to avoid using `margin-left` and `margin-right` for inner elements
```suggestion
  border-radius: 5px;
  padding: 32px 16px 16px;
```